### ISSUE TITLE
Feature request: Animated Image support

### ISSUE THREAD

**Issue by oezgan** [Tulpep/Notification-Popup-Window#11]
Hi, 
I am just a beginner but it would be nice to have animated images support. I don't even know if it is possible or not. 
Thank you very much

**Comment by ricardopolo** [comment]
I dont think that is possible, if someone finds a way to do It, I am willing to approve the PR a release a new nuget version
